src/idl_gen_cpp.cpp

```diff
@@ -1537,6 +1537,26 @@ class CppGenerator : public BaseGenerator {
     code_ += "{{PRE}}{{PARAM_TYPE}}{{PARAM_NAME}} = {{PARAM_VALUE}}\\";
   }
 
+  void GenParamNoDefault(const FieldDef &field, bool direct, const char *prefix) {
```

Comment:
seems to me this would be better as an option to the same function above, since all you want to do is not output ` = {{PARAM_VALUE}}` ?

Reply:
That's a good idea, thanks.  Done in latest commit.